crons.go

```diff
@@ -92,16 +100,28 @@ func runSentryCronsCheckin(ctx context.Context, job *batchv1.Job, eventHandlerTy
 		return errors.New("cannot find cronJob data")
 	}
 
-	// The job just begun so check in to start
-	if job.Status.Active == 0 && job.Status.Succeeded == 0 && job.Status.Failed == 0 {
-		// Add the job to the cronJob informer data
-		checkinJobStarting(ctx, job, cronsMonitorData)
-	} else if job.Status.Active > 0 {
-		return nil
-	} else if job.Status.Failed > 0 || job.Status.Succeeded > 0 {
-		checkinJobEnding(ctx, job, cronsMonitorData)
-		return nil // finished
-	}
+	hub.WithScope(func(scope *sentry.Scope) {
+
+		// If DSN annotation provided, we bind a new client with that DSN
+		client, ok := dsnData.GetClientFromObject(ctx, &job.ObjectMeta, hub.Clone().Client().Options())
```

Comment:
Do we need to do `hub.Clone()` here too?

Reply:
No, I do not believe so.